ckanext/geocat/tests/test_dataset_metadata.py

```diff
@@ -285,6 +286,12 @@ def test_documentation(self):
             ]
         )
 
+    def test_license_and_rights(self):
+        xml = self._load_xml('geocat-testdata.xml')
+        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier)
+        for resource in dataset['resources']:
+            self.assertEquals(resource['license'], "NonCommercialAllowed-CommercialAllowed-ReferenceRequired")
+            self.assertEquals(resource['rights'], "NonCommercialAllowed-CommercialAllowed-ReferenceRequired")
```

Comment:
You don't need to test this for the dataset if you're already testing it in `test_distribution_metatdata.py`, since the field is on the distribution/resource.